scripts/cli/start_cli_testnet.sh

```diff
@@ -18,11 +18,11 @@ case $1 in
         print_help;exit 0;;
     -r | --release)
         echo "Building and running client in release mode."
-        cargo run -p client --release -- $RUN_PARAMS
+        cargo run -p client --release -- $RUN_PARAMS $FLAGS
         ;;
     '')
         echo "Building and running client in debug mode."
-        cargo run -p client -- $RUN_PARAMS
+        cargo run -p client -- $RUN_PARAMS $FLAGS
```

Comment:
Also forgive me but you're going to have to change the indent on this and the other line to use a tab character, as it seems like that's the only error we're seeing in CI.

Reply:
:+1: fixed